docs/data-sources/associated_site.md

```diff
@@ -0,0 +1,72 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+# In order to regenerate this file execute `go generate` from the repository root.
+# More details can be found in the [README](https://github.com/CiscoDevNet/terraform-provider-aci/blob/master/README.md).
+subcategory: "Multi-Site"
+layout: "aci"
+page_title: "ACI: aci_associated_site"
+sidebar_current: "docs-aci-data-source-aci_associated_site"
+description: |-
+  Data source for Associated Site
+---
+
+# aci_associated_site #
+
+Data source for Associated Site
+
+## API Information ##
+
+* Class: [fvSiteAssociated](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvSiteAssociated/overview)
+
+
+* Distinguished Name Format: `uni/tn-{name}/ctx-{name}/stAsc`
+
+## GUI Information ##
+
+* Location: `Not shown in UI`
+
+## Example Usage ##
+
+```hcl
+
+data "aci_associated_site" "example_application_epg" {
+  parent_dn = aci_application_epg.example.id
+}
+
+data "aci_associated_site" "example_vrf" {
+  parent_dn = aci_vrf.example.id
+}
+
+```
+
+## Schema ##
+
+### Required ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_application_epg](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/application_epg) ([fvAEPg](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvAEPg/overview))
+  - [aci_vrf](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/vrf) ([fvCtx](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvCtx/overview))
+  - The distinquised name (DN) of classes below can be used but currently there is no available resource for it:
+    - [fvBD](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvBD/overview)
+    - [l3extInstP](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/l3extInstP/overview)
```

Comment:
We have resources for these classes. Please lookup the resource name for these classes in the legacy resource (/aci) directory. Then add them to the classes.yaml like followed:

```
fvBD:
  resource_name: "bridge_domain"